packages/office-ui-fabric-react/src/components/ContextualMenu/ContextualMenu.Props.ts

```diff
@@ -15,6 +15,10 @@ export enum ContextualMenuItemType {
   Header = 2
 }
 
+export interface IContextualMenuCallBackProps {
+  key?: string | number;
```

Comment:
shouldn't this have a reference to the item that was opened?

Also Callback is one word.